esmvaltool/utils/draft_release_notes.py

```diff
@@ -41,8 +41,8 @@
 }
 
 PREVIOUS_RELEASE = {
-    'esmvalcore': datetime.datetime(2022, 7, 15, 18),
-    'esmvaltool': datetime.datetime(2022, 7, 25, 18),
+    'esmvalcore': datetime.datetime(2022, 10, 13, 18),
```

Comment:
What about v2.7.1 ?

Reply:
To me v2.7.1 is a bugfix release and does not contain all PRs that were merged into main between Oct 13 (v2.7) and Dec 12 (v2.7.1). For example, https://github.com/ESMValGroup/ESMValCore/pull/1769. But you're right that we should make sure not to list PRs that made it into [v2.7.1](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/changelog.html#v2-7-1) for the changelog of v2.8.